iothub/service/src/DigitalTwin/readme.md

```diff
@@ -1,20 +1,91 @@
-#### Autorest config file for generating Digital Twin Service Client models and protocol layer
+*NOTE: The `DigitalTwinClient` is available only on .NET framework 4.7.2 and .NET Standard 2.0+.
```

Comment:
There is no space beween * and Note so it will just show up as *Note:......
You can either add a space or use blockquotes - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes

Reply:
I like blockquotes, I'll use that!